djagger/utils.py

```diff
@@ -151,6 +152,9 @@ def get_url_patterns(app_names: List[str]) -> List[Tuple[str, URLPattern]]:
 
             if path_app_name not in app_names:
                 continue
+        elif url_names:
```

Comment:
```suggestion
        if url_names:
```
Suggest changing `elif` to `if` conditional. In this current `elif` control flow, `url_names` will not be reached whenever `app_names` parameter is provided.

As such, in your example:
```
DJAGGER_DOCUMENT = {
    "app_names" : ['my_entire_app'],
    "url_names": ['oidc_provider_token_endpoint']
}
```
`"url_names": ['oidc_provider_token_endpoint']`   has no effect since `app_names` is also provided.

Other than this, everything else looks ok.